GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -37,6 +37,13 @@ BOOL GDTCORReachabilityFlagsContainWWAN(SCNetworkReachabilityFlags flags) {
 #endif  // TARGET_OS_IOS
 }
 
+@interface GDTCORApplication () {
+  /** Private flag to match the existing `readonly` public flag. */
+  BOOL _isRunningInBackground;
```

Comment:
We don't write it from different threads (only read), but it may make sense to add `volatile` just in case.

Reply:
or declare it as an atomic property (which is a bit more common way to declare thread-safe variables)